src/index.ts

```diff
@@ -40,9 +40,8 @@ async function asyncServerless() {
     });
 }
 
-const serverlessHandler: Promise<any> = asyncServerless();
-
 exports.handler = async (event: any = {}, context: any = {}): Promise<any> => {
+    const serverlessHandler: Promise<any> = asyncServerless();
```

Comment:
When API call to parameter store happens outside of handler function, we get the xray error. Moving the function 
 asyncServerless()  inside handler function resolves the issue.

Reply:
Traching -> tracing?  

Moving the asyncServless() call inside the handler() function will mean asyncServless() will be executed on every invocation of the lambda, right?  Is that an issue to be concerned with at all?

Reply:
good point. It would affect the performance. It should be outside.  But if it is outside then we will have xray errors

Reply:
We can leave the xray errors. The error won't stop lambda from fetching params. It just tells us xray doesn't work when it makes API calls to SSM

Reply:
It would be great if there is a way to disable xray outside of handler

Reply:
There is a [feature requirement](https://github.com/aws/aws-xray-sdk-node/issues/231) to turn of xray. But the priority is too low, and has not yet implemented.

Reply:
'AWS_XRAY_CONTEXT_MISSING' might turn off error logs at least. I will try that.

Reply:
Setting 'IGNORE_ERROR' to environment variable 'AWS_XRAY_CONTEXT_MISSING' works !